docs/commands/rhoas_service-registry_rule_list.md

```diff
@@ -13,13 +13,13 @@ rhoas service-registry rule list [flags]
 ### Examples
 
 ```
-## List global rules for artifacts of the current Service Registry instance
+## List the global rules for all artifacts in the current Service Registry instance
 $ rhoas service-registry rule list
 
-## List global rule for artifacts of a specific Service Registry instance
-$ rhoas service-registry rule list --instance-id 8ecff228-1ffe-4cf5-b38b-55223885ee00
+## List the global rules for all artifacts in a specific Service Registry instance
+$ rhoas service-registry rule list --instance-id=8ecff228-1ffe-4cf5-b38b-55223885ee00
 
-## List rule for a specific artifact
+## List the artifact-specific rules for a particular artifact
 $ rhoas service-registry rule list --artifact-id=my-artifact
```

Comment:
Suggest "List the artifact rules for a specific artifact" for minimalism. Don't think we need "artifact-specific" and "particular artifact" in the same sentence.

Reply:
Fixed in the rule.en.toml file. (Note that the docs/commands/x.md files are modular docs that are generated from the source pkg/core/localize/locales/en/cmd/y.toml files. I should have mentioned that you didn't need to review the .md files, my apologies!)

Reply:
Thanks a million, @smccarthy-ie -- I very much appreciated the speedy review!